pkg/workceptor/kubernetes.go

```diff
@@ -606,6 +648,11 @@ func (kw *kubeUnit) SetFromParams(params map[string]string) error {
 			}
 		}
 	}
+	if ked.KubePodSpec != "" && (userParams != "" || userCommand != "" || userImage != "") {
+		return fmt.Errorf("params kube_command, kube_image, kube_params not compatible with secret_kube_podspec")
```

Comment:
Probably need to check that `kw.baseParams` is also empty

Reply:
This check makes sure podspec isn't passed in alongside the other runtime options, but it's okay to have these (image, command, params) still defined in the `worker-kubernetes` definition in the yaml.

But it does make me realize that I probably should set kw.baseParams to be empty, given it won't be used at all.